metadata-ingestion/src/datahub/ingestion/source/lookml.py

```diff
@@ -452,6 +473,10 @@ def _load_viewfile(
         if self.is_view_seen(str(path)):
             return self.viewfile_cache[path]
 
+        if not self.view_pattern.allowed(pathlib.Path(path).stem.strip(".view")):
```

Comment:
Since a view file can contain multiple views and view file names are not required to be exactly the view name (even though most people use it in that manner), this way of filtering out view files seems unsafe. WDYT?

Reply:
The root reason for the PR is we (and we also expect for other users) have some legacy looker view files that have some syntax errors in it, so we don’t want datahub to read these view files.
I understand that within a model file, a view name does not necessarily be the same as view file, but “forbidding datahub to read the view file” seems the most direct way to meet the needs.

Per out discussion, a specific` file_ allow deny` pattern would be better. May I ask if it's something like view_file_pattern , besides the existing view_pattern ?